CHANGELOG.md

```diff
@@ -15,6 +15,9 @@
   * If you previously manually `kubectl apply`'d secrets that are not passed to kubernetes-deploy, your first deploy using this version is going to delete them.
   * If you previously passed secrets manifests to kubernetes-deploy and they are no longer in the set you pass to the first deploy using this version, it will delete them.
 
+*Other*
+
+- The secret `ejson-keys` will never be pruned by kubernetes-deploy. Instead, it will remove the `kubectl.kubernetes.io/last-applied-configuration` annotation if it is present and log a warning. This _only_ affects the `ejson-keys` secret. All other secrets are subject to pruning as normal. ([#447](https://github.com/Shopify/kubernetes-deploy/pull/447))
```

Comment:
Since pruning is first introduced in this version and is described above, this isn't really a change. Maybe this should be one of the `*` sub-points of the feature above?